tor/strings/en_US.yml

```diff
@@ -90,6 +94,8 @@ responses:
       Thanks for letting me know! It looks like the content to transcribe has been removed, so I removed the post on our side. Cheers!
     post_already_completed: |
       I can't process an unclaim on a post that's already been completed!
+    claimed_other_user: |
+      It appears that you have not claimed this post!
```

Comment:
This message makes it sound like they just forgot to claim it, not that someone else claimed it. Let's change this to something like "Sorry, this post has been claimed by someone else."